src/applications/personalization/dashboard-2/tests/e2e/Dashboard.cypress.spec.js

```diff
@@ -0,0 +1,61 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import { MY_VA_ROOT } from 'applications/personalization/dashboard-2/constants';
+import { mockUser } from '@@profile/tests/fixtures/users/user.js';
+
+const mockFeatureToggles = () => {
+  cy.route({
+    method: 'GET',
+    status: 200,
+    url: '/v0/feature_toggles*',
+    response: {
+      data: {
+        features: [
+          {
+            name: 'dashboard_show_dashboard_2',
+            value: true,
+          },
+        ],
+      },
+    },
+  });
+};
+
+/**
+ *
+ * @param {boolean} mobile - test on a mobile viewport or not
+ *
+ * This helper:
+ * - loads the my VA Dashboard,
+ *   checks that focus is managed correctly, and performs an aXe scan
+ */
+function checkAllPages(mobile = false) {
+  mockFeatureToggles();
+  cy.visit(MY_VA_ROOT);
+
+  if (mobile) {
+    cy.viewport('iphone-4');
+  }
+
+  cy.findByTestId('dashboard-title').should('exist');
```

Comment:
since we are testing that this thing has focus in the next assertion, this isn't really needed